scripts/giphy.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var request = require('request');
+
+function spotify(bot, controller, message) {
```

Comment:
This function isn't the same name as the command you've specified below. It should be 
```js
function giphy(bot, controller, message) {
// your code here
}
```